### PR TITLE
feat: add Penpot Desktop

### PR DESCRIPTION
**Pull request by KorbsStudio**
### Checklist

- [x] I have read and completed the checklist of [Submission Guide](https://github.com/electron/apps/blob/master/contributing.md#submission-guidelines)
- [x] My application is open-source and has been created for at least 20 days
- [ ] My application is closed-source and here is a proof it uses Electron:

### Description
Penpot Desktop on Linux

Penpot Desktop delivers a desktop-like experience for Penpot users with the additional of integrating tabs to conveniently traverse back and forth between projects. Offline support is available through the select your own instance option in settings, as well as the theme settings that may be applied to either the entire desktop app or simply the Penpot dashboard.  

This "unofficial" software has also been recommended by Penpot's CEO on their [forum](https://community.penpot.app/t/is-there-a-desktop-app-option-for-penpot/2038) as a current alternative to getting Penpot on desktop.

[Source Code](https://code.korbsstudio.com/KorbsStudio/Penpot-Desktop)

### CODE REVIEW

**Review by erickzhao — APPROVED** [review]

File: apps/penpot-desktop/penpot-desktop.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Penpot Desktop
+description: A Desktop-Like Experience
```
Comment (erickzhao) [inline apps/penpot-desktop/penpot-desktop.yml:2]: This sounds a bit weird as a description 🤔

Comment (KorbsStudio) [reply apps/penpot-desktop/penpot-desktop.yml:2]: Okay dokie! I've got something else I use on Product Hunt. Lemme slap it on there.

**Comment by BlackHole1** [comment]
CI failed. Can you fix it?

**Comment by KorbsStudio** [comment]
I've fixed the issue with the CI failing. I've renamed the icon file and made sure the description doesn't start with the app's name.

**Comment by KorbsStudio** [comment]
Is there something missing?

**Review by BlackHole1 — APPROVED** [review]

**Pull request merged**